src/Microsoft.AspNetCore.Razor.Evolution/DirectiveTokenDescriptor.cs

```diff
@@ -6,7 +6,5 @@ namespace Microsoft.AspNetCore.Razor.Evolution
     public class DirectiveTokenDescriptor
```

Comment:
Btw, this feels a bit weird now. It's a class with one enum field. Do we plan on sticking more things on this?